packages/edit-site/src/components/header/index.js

```diff
@@ -49,16 +49,22 @@ export default function Header( {
 			getTemplatePartId,
 			getTemplateType,
 			getPage,
-			getShowOnFront,
 		} = select( 'core/edit-site' );
+
+		// eslint-disable-next-line camelcase
```

Comment:
Maybe there's a convention that we're really supposed to return camel case from the endpoint here?

Reply:
The existing data stored here doesn't seem to be in camel case:

<img width="404" alt="Screen Shot 2020-07-13 at 11 22 43 AM" src="https://user-images.githubusercontent.com/6265975/87339253-326c1180-c4fb-11ea-9482-5510bbfc99ca.png">

I'm not sure what API convention is here, though. At the very least, I think it needs to use underscores when passed into the API, but I don't know when it needs to be converted

Reply:
I've renamed the variable inline. I think that's good enough